subworkflows/local/quilt_subworkflows/impute_quilt.nf

```diff
@@ -0,0 +1,54 @@
+include { QUILT_QUILT     } from '../../../modules/nf-core/quilt/quilt/main'
+
+workflow IMPUTE_QUILT {
+
+    take:
+    ch_hap_legend                            // channel: [ val(meta),hap, legend ]
+    ch_input                                //  channel: [ val(meta),bam, bai ]
+    ch_chunks                              //   channel: [ val(meta), start_coordinate, end_coordinate, number ]
+
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    posfile = []
+    phasefile = []
+    posfile_phasefile = [[id: null], posfile, phasefile]
+    genetic_map_file = []
+    fasta = [[id:'test'], []]
+
+    ngen = params.ngen
+    buffer = params.buffer
+
+    ch_bam_bamlist = ch_input
+
+    if (genetic_map_file.isEmpty()) {
+        ch_hap_chunks = ch_hap_legend.combine(ch_chunks, by:0).map { it + ngen + buffer + [[]] }
```

Comment:
Why do you use combine instead of join here ?

Reply:
There are cases where there are going to be many chunks per chromosome.